app/poi_api/api.py

```diff
@@ -1,32 +1,32 @@
-from .poi_service import POI_Service
+from .service import list_POI, get_details_for_POI, new_POI_suggestion
 from flask import jsonify
-from .poi_errors import POINotFoundError, InvalidPOISuggestionError
-
-poi_service_class = POI_Service()
+from .errors import POINotFoundError, InvalidPOISuggestionError
+from typing import Dict
 
 # TODO Exception handling

```

Comment:
For all endpoints where you want to get a User, you should be using the with_auth_user decorator. Even if you don't explicitly need to know who the user is, if you don't use the decorator connexion still expects all functions to have a token_info parameter (or **kwargs, which is probably preferred since you're not using token_info, so no need to explicitly take in that keyword argument)